src/hdusd/config.py

```diff
@@ -16,6 +16,9 @@
 
 logging.limit_log('', level_show_min=logging.INFO)
 
+matlib_url = "https://matlibapi.cistest.luxoft.com/api"
+engine_use_preview = True
```

Comment:
Should these vars be capitalized (since they are const)?

Reply:
not here